components/tools/OmeroWeb/test/integration/weblibrary.py

```diff
@@ -90,6 +92,8 @@ def _csrf_post_response(django_client, request_url, data, status_code=200):
 def _get_response(django_client, request_url, query_string, status_code=405):
     query_string = urlencode(query_string.items())
     response = django_client.get('%s?%s' % (request_url, query_string))
+    if response.status_code != status_code:
+        print response
```

Comment:
:+1: for the extra debugging info, for clarity, this can be coupled with the `assert` using

```
assert response.status_code == status_code, response
```